pkg/configuration/config_manager/config_handler.go

```diff
@@ -232,7 +274,7 @@ func (s *shellCommandHandler) VolumeHandle(ctx context.Context, event fsnotify.E
 	}
 
 	logger.Info(fmt.Sprintf("updated parameters: %v", updatedParams))
-	if err := s.execHandler(ctx, updatedParams, args); err != nil {
+	if _, err := s.execHandler(ctx, updatedParams, args); err != nil {
```

Comment:
If ret is not used, it should not be returned.

Reply:
Fixed in bbf86cdffc028e83746c42b7fa33d25479c37ec0